### DISCUSSION TITLE
Thoughts on the Approach in EllSeg

### DISCUSSION THREAD

**Discussion by cancan101** [openPupil/Open-PupilEXT#48] [Q&A]
I am curious if the model proposed in the [EllSeg paper](https://arxiv.org/abs/2007.096000) was investigated?

slightly more to date work from the same team with a similar approach can be found here: https://bitbucket.org/RSKothari/multiset_gaze/src but is designed to consume multiple datasets for jointly training on multiple sources of data.

**Comment by BZandi** [comment]
The algorithm was not investigated in our work. If you like you can create a pull request and add this algorithm to PupilEXT. We provide a community branch for PupilEXT, allowing to add new features via pull requests (see here: https://github.com/openPupil/Open-PupilEXT/tree/Experimental-Community-Version)

**Comment by kheki4** [comment]
Hi there @cancan101 . Me and a friend of mine @sasfog are keen on adding new features to the application. We are working on the branch Babak also mentioned. We are currently finishing a new set of features, and testing stability. Once we are done with that, we will prepare a new release in the summer. After that, we will head to new directions: try to add entry-level gaze detection, and possibly try new pupil detection algorithms too. But it is too soon to say dates or exact priorities for now. Thanks for your idea! Please stay tuned for updates :)
